.bra.toml

```diff
@@ -1,6 +1,7 @@
 [run]
 init_cmds = [
   ["GO_BUILD_DEV=1", "make", "build-go"],
+  ["GO_BUILD_DEV=1", "make", "gen-go"],
```

Comment:
this should be first in both arrays, it needs to be run before build-go to be useful